### ISSUE TITLE
Email forward not working

### ISSUE THREAD

**Issue by iaxat** [mozilla/fx-private-relay#2001]
The email forward feature is not working and none of the aliases' emails are getting forwarded. It says there is a soft bounce, and it will deliver again, but the thing is all other emails directly to that email are working just fine.
I am kind of worried because of this issue, there are quite a few places where the email for communication is among the Firefox relay.

**Comment by PabloDiehl** [comment]
I'm facing the same problem `We are currently unable to send email to <mail>. We received a ⁨soft⁩ “bounce” from your email provider when trying to forward emails to you.`, I'm using Yahoo mail by the way.

**Comment by tcinotto** [comment]
@iaxat Thank you for the report and we're sorry to hear Relay isn't working for you. Could please share what email domain you are using? cc: @jwhitlock could you take a look into this? 

@PabloDiehl Thank you for the report as well. We'll take a look at this with our engineering team. For you, how frequently is this happening?

**Comment by PabloDiehl** [comment]
Hello @tcinotto, thanks for the reply.
The first time I noticed this problem was on May 27th.

**Comment by Max-F-Helm** [comment]
I have the same issue with my "@yahoo.com" domain. No new Mails could be forwarded.

**Comment by iaxat** [comment]
Yahoo

**Comment by iaxat** [comment]
I am not even getting GitHub updates, because I use relay for that as well.

**Comment by say-yawn** [comment]
Hello, we looked at this issue and did not find anything on our end that could have caused this. We saw that there was a spike of emails being bounced from `yahoo` domains so we have reached out to them and they have made the update to which we are seeing as decreased bounces from Yahoo. Thanks for reporting the problem and feel free to open this issue again if you are experiencing problems on email forwarding.
